c20_game_of_life/game_of_life.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+type universe [][]bool
+
+// Creates a new instance of a universe - every cell is initialized to false
+func newUniverse(height int, width int) universe {
+	h := make([][]bool, height)
```

Comment:
That's one way of looking at it. I saw it initially as it's just the slice of slices, but the slices aren't filled in yet, so it's just the height. For the sake of clarity, it might make more sense to call it "u"